api/src/api/taxFilingRouter.test.ts

```diff
@@ -19,6 +19,7 @@ describe("taxFilingRouter", () => {
 
   let stubUserDataClient: jest.Mocked<UserDataClient>;
   let apiTaxFilingClient: jest.Mocked<TaxFilingInterface>;
+  let stubTaxEncryptionDecryptionClient: jest.Mocked<EncryptionDecryptionClient>;
```

Comment:
A nitpick, but technically this is just a `stubEncryptionDecryptionClient` (no "Tax") because there's nothing in the abstract interface (nor the implementation, even) that makes it specific to tax. It can encrypt and decrypt anything.